### PR TITLE
BWS Fix: run emailService

### PR DESCRIPTION
**Pull request by Gamboster**

### CODE REVIEW

**Comment by matiu** [comment]
needs rebase.

**Comment by Gamboster** [comment]
@matiu Rebase done!

File: packages/bitcore-wallet-service/src/fiatrateservice/fiatrateservice.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env node
-const config = require('../config');
+
+var config = require('../config');
```
Comment (matiu) [inline packages/bitcore-wallet-service/src/fiatrateservice/fiatrateservice.ts]: why no `const`?

File: packages/bitcore-wallet-service/src/lib/emailservice.ts [hunk 2]
```diff
@@ -93,17 +93,17 @@ export class EmailService {
       });
     };
 
-    this.defaultLanguage = opts.emailOpts.defaultLanguage || 'en';
-    this.defaultUnit = opts.emailOpts.defaultUnit || 'btc';
+    this.defaultLanguage = opts.emailOpts && opts.emailOpts.defaultLanguage ? opts.emailOpts.defaultLanguage : 'en';
```
Comment (matiu) [inline packages/bitcore-wallet-service/src/lib/emailservice.ts]: maybe add:

`opts.emailOpts = opts.emailOpts || {};`

 before all this checks, to simplify all the following lines?

File: packages/bitcore-wallet-service/src/fiatrateservice/fiatrateservice.ts (hunk 1 above)
Comment (Gamboster) [reply packages/bitcore-wallet-service/src/fiatrateservice/fiatrateservice.ts]: For some reason, if I use `const` I get a typescript error .. something like "You can not declare config 2 times", taking into account the config of fiatrateservice and emailservice
I'm not sure why it compares the declaration of 2 different files

File: packages/bitcore-wallet-service/src/lib/emailservice.ts (hunk 2 above)
Comment (Gamboster) [reply packages/bitcore-wallet-service/src/lib/emailservice.ts]: Yes. Done!

**Pull request merged**